plugins/node/opentelemetry-koa-instrumentation/package.json

```diff
@@ -0,0 +1,74 @@
+{
+  "name": "@opentelemetry/koa-instrumentation",
+  "version": "0.9.0",
+  "description": "OpenTelemetry Koa automatic instrumentation package.",
+  "main": "build/src/index.js",
+  "types": "build/src/index.d.ts",
+  "repository": "open-telemetry/opentelemetry-js-contrib",
+  "scripts": {
+    "test": "nyc ts-mocha -p tsconfig.json 'test/**/*.ts'",
+    "codecov": "nyc report --reporter=json && codecov -f coverage/*.json -p ../../",
+    "tdd": "yarn test -- --watch-extensions ts --watch",
+    "clean": "rimraf build/*",
+    "lint": "eslint . --ext .ts",
+    "lint:fix": "eslint . --ext .ts --fix",
+    "precompile": "tsc --version",
+    "version:update": "node ../../../scripts/version-update.js",
+    "compile": "npm run version:update && tsc -p .",
+    "prepare": "npm run compile"
+  },
+  "keywords": [
+    "opentelemetry",
+    "koa",
+    "nodejs",
+    "tracing",
+    "profiling",
+    "plugin",
+    "instrumentation"
+  ],
+  "author": "OpenTelemetry Authors",
+  "license": "Apache-2.0",
+  "engines": {
+    "node": ">=8.0.0"
+  },
+  "files": [
+    "build/src/**/*.js",
+    "build/src/**/*.d.ts",
+    "doc",
+    "LICENSE",
+    "README.md"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "devDependencies": {
+    "@koa/router": "^9.3.1",
+    "@opentelemetry/context-async-hooks": "0.10.2",
+    "@opentelemetry/node": "0.10.2",
+    "@opentelemetry/tracing": "0.10.2",
```

Comment:
An open question/thought having dug through this stuff recently (to everyone, not necessarily the author)...

It seems to be a pattern to have these dev-dependencies locked at a single version while the regular deps do the typical ^version. That has recently caused issues and seems like something that would be regularly prone to, at least pre-major release. 

Why wouldn't we want these dev-deps to update per usual and catch things and/or automatically keep in sync with full publishes of everything in opentelemetry-js? Wouldn't we want to know if these plugins were broken from a minor/patch release in the future?

Reply:
Great point for discussion! I'm interested in hearing feedback from everyone else

Reply:
I could be wrong, but I think dependabot intentionally locks them to a single version and updates on its own PRs only